ci/README.md

```diff
@@ -0,0 +1,110 @@
+
+### Unit Tests
+
+Unit tests are executed via `rake` and are contained in [spec/haproxy/templates](../spec/haproxy/templates).
+
+### Acceptance Tests
+
+The acceptance tests run a full BOSH director and exercise creating and running the candidate `haproxy-boshrelease` against a test suite that covers a wide range of features and use cases supported by it.
+
+The code can be found in [acceptance-tests](../acceptance-tests/).
+
+The `haproxy-boshrelease` is deployed via the manifest defined in [manifests/haproxy.yml](../manifests/haproxy.yml). Most of the tests use BOSH ops-files to modify this manifest before running it.
+
+The deployed HAProxy will in most cases have a functioning backend that simply responds with `Hello cloud foundry` to HTTP requests. This backend can be omitted and will lead to a failure state, as HAProxy in its current configuration requires the backend to start.
+
+#### Writing new Acceptance Tests
+
+There are examples for various types of tests already in the source code. Those include startup and draining behaviour, various types of requests and specific configurations where HAProxy modifies the request as well as general functionality checks to avoid regressoins.
+
+ There are a few things to highlight when developing new acceptance tests:
+
+1. The HAProxy deployed via the release is run in a container. The port to HAProxy and to the backend are forwarded via SSH tunnel to the test runner and allow interacting with either of those servers.
+2. The HAProxy deployment is carried out by:
+
+   ```golang
+   func deployHAProxy(baseManifestVars baseManifestVars, 
+                      customOpsfiles []string, 
+                      customVars map[string]interface{}, 
+                      expectSuccess bool) (haproxyInfo, varsStoreReader) { ... }
+   ```
+
+   Using `expectSuccess`, the boshrelease can be started with an incomplete configuration and manipulated by the test. This is useful, when additional resources are referenced in the HAProxy configuration and need to be uploaded to the container where HAProxy runs. Because its address is only known once BOSH deploys this container, it is not possible to upload files ahead of time.
+3. Use ginkgo's "Focus" feature to execute a single test instead of the complete acceptance test suite by adding `F` to the `Declare` statements in a test, i.e. `FDeclare(...)` vs. `Declare(...)`.
+4. Most tests involve ops-files, which modify the deployment manifest. Those ops-files can be tested locally via BOSH, using:
+
+   ```shell
+   bosh interpolate manifests/haproxy.yml --ops-file opsfile.yml
+   ```
+
+   This command will output (but not overwrite) the resulting HAProxy manifest after the ops-file has been applied. This is also the most convenient way to ensure that the syntax and functionality in the ops-file are correct and can be handled by the BOSH CLI.
+
+#### Running Acceptance Tests Locally
```

Comment:
Could you please mention if it is only one option to run the acceptance tests via PR and concourse job acceptance-test-pr?

Reply:
will do.